metron-deployment/development/centos6/docker_run_ansible.sh

```diff
@@ -14,15 +16,25 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #
-[defaults]
-host_key_checking = False
-library = ../extra_modules
-roles_path = ../roles
-pipelining = True
-remote_user = centos
-forks = 20
-log_path = ./ansible.log
-
-# fix for "ssh throws 'unix domain socket too long' " problem
-[ssh_connection]
-control_path = ~/.ssh/ansible-ssh-%%h-%%r
+
+
+#
+# This script runs IN the docker container
+#
+
+cd /root/metron || exit 1
+
+# make sure we have the right c++ tools
+
+#shellcheck disable=SC1091
+source /opt/rh/devtoolset-6/enable
+
+sed -i -e '/^node1.*/d' ~/.ssh/known_hosts
```

Comment:
It my test run, I am getting this error.  
```
sed: can't read /root/.ssh/known_hosts: No such file or directory
```

I understand this won't be a problem on the second run, but maybe we can guard this with a file existence check.